src/libraries/System.Private.CoreLib/src/Internal/Console.Android.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Internal
 {
-    public static partial class Console
+    internal static partial class Console
```

Comment:
This makes the Console class rather useless. I don't know if it even makes sense to keep it if it's internal.

One of the use cases for this is to print stuff from test programs. I typically use it when I have a small repro *.cs program that I compile with `csc /noconfig /nostdlib myrepro.cs /r:System.Private.CoreLib.dll` and then just `corerun myrepro.exe` - it allows to run managed code without having to build a single project from src\libraries or setting up CORE_ROOT or any of those chores.

Reply:
That's a nice trick.